README.md

```diff
@@ -18,6 +18,17 @@ API_URL = "https://iscsc.fr/api/articles"
 ```
 
 ## Start the bot
-To run it on the host system simply `python3 main.py`.
+### Directly on host system
+To run it on the host system simply run `python3 main.py`.
 
-**But** you really should containerize it thanks to the provided `Dockerfile`, you just have to run `./run.sh` the script clean previous running containers, build the current one and start it.
+:warning: but you **should really** consider using the [containerized deployment](#Containerized-with-Docker).
+
+### Containerized with Docker
+The bot can run in Docker thanks to the provided `Dockerfile`. To ease the starting, you just have to run `./run.sh`: the script cleans previous running containers, builds the current one and starts it.
```

Comment:
Phrasing is weird here "To ease the starting", maybe just say "To start the bot in docker, [...]"

Reply:
you're right don't know what I was thinking ahah

Reply:
fixed [bf05de2](https://github.com/iScsc/iscsc.fr-blog-notify/pull/4/commits/bf05de2dd780769f249ef691f72c233f63ba0bf9)